vignettes/customise.Rmd

```diff
@@ -333,7 +333,7 @@ For example, the following yaml adds a new "twitter" component that appears to t
 ``` yaml
 navbar:
   structure:
-    right: [twitter, github]
+    right: [search, twitter, github]
```

Comment:
You'd also need to add `lightswitch` here, if you're using the light switch.

Reply:
Agreed! better to mention it as specifying `right` has the "power" to make you loose `search` and `lightswitch` if not added explicitly?

Reply:
Maybe it's worth adding a sentence to that effect? Something like: The downside of customising that navbar in this way is that you have to specify the other components that are specified by default, and if pkgdown later changes the defaults, you'll have to update you `_pkgdown.yml`.

Reply:
I added the note elsewhere. Let me know what you think